packages/grid_client/src/clients/tf-grid/balances.ts

```diff
@@ -16,6 +17,13 @@ class TFBalances extends Balances {
     const decimalAmountInTFT = decimalAmount.mul(10 ** 7).toNumber();
     return await super.transfer({ address: options.address, amount: decimalAmountInTFT });
   }
+
+  async getMoreFunds(address: string, callback: any) {
```

Comment:
this method should work with 0 args. we already know the caller's address and let's find a way to not send the callback to it and handle it internally.

- I suggest to create an instance from the tfchain client with `Alice` as mnemonic (it will fail in validation so let's make it as exception).
- Then use this client to do the transfer.